### PR TITLE
Include custom block styles from plugin

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
_No ticket needed_

[Demo page](https://www-dev.greenpeace.org/test-nix/register-block-styles/)

### Required by
- https://github.com/greenpeace/planet4-master-theme/pull/2258
- https://github.com/greenpeace/planet4-master-theme/pull/2259

Copied from [editor.js](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/admin/js/editor.js#L5C3-L86C6)

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [nix](https://www-dev.greenpeace.org/test-nix) | [admin](https://www-dev.greenpeace.org/test-nix/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-nix/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-nix) | [composer-local.json](https://github.com/greenpeace/planet4-test-nix/blob/main/composer-local.json)

:watch: 2024.05.15 13:48:40

File: assets/src/block-styles/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+
+const {unregisterBlockStyle, registerBlockStyle} = wp.blocks;
+const {select} = wp.data;
+const {__} = wp.i18n;
+
+export const registerBlockStyles = () => {
+  // Remove Take Action and Campaign covers styles for Covers block in campaigns
```
Comment (mleray) [inline assets/src/block-styles/index.js:7]: I'm not sure we'll be moving the Covers block to the theme (potentially replacing it with the Actions List), so maybe we shouldn't move this code?

**Review by mleray — APPROVED** [review]
Just a small comment but it's not a blocker - looks good to me 👍

File: assets/src/block-styles/index.js (hunk 1 above)
Comment (GP-Dan-Tovbein) [reply assets/src/block-styles/index.js:7]: Make sense for me!

**Pull request merged**